### DISCUSSION TITLE
Unable to set hotkey to Ctrl-WIN-Space?

### DISCUSSION THREAD

**Discussion by jafin** [Flow-Launcher/Flow.Launcher#913] [Q&A] [answered]
If i attempt to set the hotkey as Ctrl-Win-Space it sets it Ctrl-Win-LWin.

Is this supported?

**Comment by taooceros** [comment, marked as answer]
I think win is not supported for now

**Reply by taooceros** [reply]
Win + Space may not be available due to hotkey conflict. It is used for changing input language.

**Reply by jafin** [reply]
![image](https://user-images.githubusercontent.com/127927/146719048-1dfc1eb6-096f-4499-aa27-8d20bbe6d014.png)

In Win Powertoys Run the combination of `Ctrl-WIN-Space` is acceptable.  So I assume just a limitation in this app at present. I don't believe it is a system level conflict.

**Reply by onesounds** [reply]
> ![image](https://user-images.githubusercontent.com/127927/146719048-1dfc1eb6-096f-4499-aa27-8d20bbe6d014.png)
> 
> In Win Powertoys Run the combination of `Ctrl-WIN-Space` is acceptable.  So I assume just a limitation in this app at present. I don't believe it is a system level conflict. 

Weird Do you use multiple input method?

If you don't use multilange and work normally in powertoy, we have a problem. We are discussing implementing a function to replace Winkey, and the general shortcut is correct only when it works. Currently, there are parts that do not work out as you say. but in this case, I tested powertoy for the answer and checked It doesn't works.  (It seems to be possible, but it doesn't works for me) 

anyway, OK. We'll check the detail.

**Reply by taooceros** [reply]
Yes powertoy uses a more low level hotkey model than us. Therefore, they can support more hotkey.

**Comment by onesounds** [comment]
In this case, Win+Space is reserved for OS and does not work properly. 
We will add indicate later that these hotkeys cannot be set later.

**Reply by gxcodes** [reply]
I don't see any OS reservation for Win+Space, have used this shortcut with Wox (in powertoys run) previously without any problem.
